index.dd

```diff
@@ -123,6 +123,21 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Print hex dump,
+----
+void main()
+{
+    import std.algorithm, std.stdio, std.file;
+    enum cols = 16;
+    thisExePath.File("rb").byChunk(cols).each!(chunk =>
+        writefln!"%(%02X %)%*s  %s"(
```

Comment:
This code depends on the magic of formatted write, which is obscure and arcane. It could use some comments and/or a link to the writefln docs.